demos/github-demo/src/isograph-components/RepositoryLink.tsx

```diff
@@ -1,33 +1,29 @@
-import { iso } from '@isograph/react';
+import { iso } from "@isograph/react";
 
-import type { ResolverParameterType as RepositoryLinkParams } from '@iso/Repository/RepositoryLink/reader.isograph';
+import { Link } from "@mui/material";
 
-import { Link } from '@mui/material';
-
-export const RepositoryLink = iso<RepositoryLinkParams>`
+export const RepositoryLink = iso(`
```

Comment:
Overloads only work when we pass a string, not use iso with a template literal, so we'll have to convert everything to this syntax